types/three/test/renderers/renderers-renderTarget-texture2DArray.ts

```diff
@@ -0,0 +1,243 @@
+import * as THREE from 'three';
```

Comment:
Where is this test file coming from? Is this test file why we're making the parameters in the constructors optional? It may be correct, but I'm suspicious since it's not documented that these parameters are optional, and the code doesn't default those properties to anything.

Reply:
No sorry, I had a discussion on the `three` repo about the WebGLRenderer optional params, I need to edit this example, but its taken from the repo, it's about testing `setTexture` in `WebGLRenderTarget`